### ISSUE TITLE
NTR: Peripheral nerve biopsy: metachromasia (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9129]
New term request Peripheral nerve biopsy: metachromasia (segal_100513184444):

**Comment by pnrobinson** [comment]
@MickeySegal
does the following capture the desried context?

Term: Peripheral nerve metachromasia
Def: A type of metachromasia localized to a peripheral nerve. Metachromasia may be defined as the staining of tissue or tissue components such that the color of the tissue-bound dye complex differs significantly from the color of the original dye complex to give a marked contrast in color. For instance in  metachromatic leukodystrophy, bloated Schwann cells, containing granularmetachromatic storage material May display a characteristic yellow green dichroism under crossed polarised light.
Comment See Figure 2 of PMID:16291896.

**Comment by MickeySegal** [comment]
Yes

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001030
- label: Peripheral nerve metachromasia
- parents: [Abnormal peripheral nervous system morphology (HP:0000759)](https://hpo.jax.org/app/browse/term/HP:0000759)
- pmid: PMID:16291896
- def: A type of metachromasia localized to a peripheral nerve. Metachromasia may be defined as the staining of tissue or tissue components such that the color of the tissue-bound dye complex differs significantly from the color of the original dye complex to give a marked contrast in color. For instance in metachromatic leukodystrophy, bloated Schwann cells, containing granularmetachromatic storage material May display a characteristic yellow green dichroism under crossed polarised light.
- comment: See Figure 2 of PMID:16291896.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9129
- synonyms:

**Comment by MickeySegal** [comment]
I see we also added https://hpo.jax.org/browse/term/HP:6000079 in
https://github.com/obophenotype/human-phenotype-ontology/issues/9241

They seem like much the same thing.  Do we need both?

**Comment by pnrobinson** [comment]
check this

**Comment by pnrobinson** [comment]
merged. 
The main term is  https://hpo.jax.org/browse/term/HP:6000079
